startx: add page (#8752)
tldr-pages/tldr: pages/linux/startx.md

# startx

> > The startx script is a front end to xinit that provides a somewhat nicer user interface for running a single session of the X Window System.

Reviewer: ```suggestion
> The startx script is a front end to xinit that provides a nice user interface for running a single session of the X Window System.
```
Can we rephrase this a bit?
> Even though the man page has this description.

> More information: <https://x.org/releases/X11R7.5/doc/man/man1/startx.1.html>.

- Start an X session:

`startx`

- Start an X session with a predefined depth value:

> `startx -- -depth {{number}}`

Reviewer: ```suggestion
`startx -- -depth {{value}}`
```
I think the value would be better in this command argument.

- Start an X session with a predefined dpi value:

> `startx -- -dpi {{number}}`

Reviewer: ```suggestion
`startx -- -dpi {{value}}`
```

> - Override your `.xinitrc` settings and start a new X session:

Reviewer 2: We try to avoid `you`/`your`
```suggestion
- Override the settings in the `.xinitrc` file and start a new X session:
```

> - Override your .xinitrc settings and start a new X session:

Reviewer: ```suggestion
- Override your `.xinitrc` settings and start a new X session:
```

> `startx /path/to/{{window_manager | desktop_environment }}}`

Reviewer: How about this?
```suggestion
`startx /{{path/to/window_manager_or_desktop_environment}}`
```
Similar to [path/to/file_or_directory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

Or
```suggestion
`startx /{{path/to/window_manager | desktop_environment}}`
```

@mfrw @CleanMachine1 @navarroaxel What do you think about this?
